Update fork test instructions (#442)
liberland/liberland_substrate: substrate/scripts/fork-test/README.md

1. Check current testnet version.
2. Checkout repository for this version. For example, if testnet has runtime in version `v27.0.0`, checkout tag `v27.0.0`

> 3. Build binary with embedded testnet runtime: `cargo build --release -F testnet-runtime --features metadata-hash`:

Reviewer: ```suggestion
3. Build binary with embedded testnet runtime: `cargo build --release --features testnet-runtime --features metadata-hash`:
```

  * copy it to tmp to make it easier: `cp ./target/release/substrate-node ./substrate/scripts/fork-test/liberland-fork-substrate/tmp/liberland-testnet-v27.0.0`
4. Download runtime you want to test from [GitHub Releases](https://github.com/liberland/liberland_substrate/releases)
  * copy it to tmp to make it easier:  `cp bastiat-v28.0.0.wasm ./liberland-fork-substrate/tmp/bastiat-v28.0.0.wasm`
  * if building runtime yourself,

>     * Checkout the branch with the newer version to test (v28 in this example)and build it `cargo build --release -F testnet-runtime --features metadata-hash`

Reviewer: ```suggestion
    * Checkout the branch with the newer version to test (v28 in this example)and build it `cargo build --release --features testnet-runtime --features metadata-hash`
```

Just to keep it coherent for readability (`-F` is a short version of `--features`)

>     * Checkout the branch with the newer version to test (v28 in this example)and build it `cargo build --release --features testnet-runtime --features metadata-hash`

Reviewer 2: How about going straight for srtool?

```suggestion
    * Checkout the branch with the newer version to test (v28 in this example)and build it `env SRTOOL_TAG=1.81.0 RUNTIME_DIR=./substrate/bin/node/runtime PACKAGE=kitchensink-runtime BUILD_OPTS='"-F testnet-runtime,metadata-hash"' srtool build`
```

Reviewer 2: The path to wasm will be different - it will be in `substrate/bin/node/runtime/target/srtool/release/wbuild/kitchensink-runtime/kitchensink_runtime.compact.compressed.wasm`

1. Check current mainnet version.
2. Download the binary with the same version as one running currently on mainnet from [GitHub Releases](https://github.com/liberland/liberland_substrate/releases)
   * copy it to tmp to make it easier: `cp linux_x86_build /tmp/liberland-mainnet-v27.0.0`

>    * if building binary yourself `cargo build --release -F --features metadata-hash`

Reviewer: ```suggestion
   * if building binary yourself `cargo build --release --features metadata-hash`
```

   * `cp ./target/release/substrate-node ./substrate/scripts/fork-test/liberland-fork-substrate/tmp/liberland-mainnet-v27.0.0`
4. Download runtime you want to test from [GitHub Releases](https://github.com/liberland/liberland_substrate/releases) - it will be a different version than the binary!
   * copy it to tmp to make it easier:  `cp mainnet-v28.0.0.wasm /tmp/mainnet-v28.0.0.wasm`
   * if building runtime yourself,

>      * Checkout the branch with the newer version to test (v28 in this example)and build it `cargo build --release -F --features metadata-hash`

Reviewer: ```suggestion
     * Checkout the branch with the newer version to test (v28 in this example)and build it `cargo build --release --features metadata-hash`
```
